docs/aws/nRFConnectForCloudAPI.rst

```diff
@@ -0,0 +1,55 @@
+.. _aws-nrf-connect-for-cloud-api:
+
+nRF Connect for Cloud API
+#########################
+
```

Comment:
You might want to have this added: 
 .. contents::
   :local:
   :depth: 2

Reply:
Btw, PMT has decided to change the product name: nRF Connect for Cloud -> nRF Cloud, but not before NCS 1.6.0.

Reply:
I didn't add it because it's a very short page.

Reply:
Fair enough.